interview-questions.md

```diff
@@ -7,60 +7,61 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: git is the command line commands used for source control. github is the internet cloud storage space where git stores the data/information
 
-  Researched answer:
+  Researched answer: Github is the internet cloud storage facility where the source control code is kept. Git is the version control system that lets you manage source code history.
 
 
 
 2. What types of evaluations will return a Boolean value?
 
-  Your answer:
+  Your answer: Logical operators (>, <, =, &&, ||)
```

Comment:
Logical operators are just `&&` and `||` 
And a `=` is an assignment operator
You have the right idea, just missing a couple details.